### PR TITLE
Deploy Kube-Prometheus-Stack

### PR DESCRIPTION
**Pull request by ldb**
Yes, we had to import the whole chart >.>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
 
 # Bootstraping
 
-This repository is (mostly) fully automated. It follows ArgoCD's "[App-of-Apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/)" pattern, albeit with small differences.
```
Comment (ldb) [inline README.md:45]: This was by mistake

File: linkerd-viz/values.yaml [hunk 2]
```diff
@@ -1,5 +1,16 @@
-linkerd-viz: {}
-  # grafana:
-  #   enabled: false
-  # prometheus:
-  #   enabled: false
\ No newline at end of file
+linkerd-viz:
```
Comment (akkbng) [inline linkerd-viz/values.yaml:1]: Should we apply the "values-common.yaml, values-dev.yaml, values-live.yaml" structure to linkerd-viz deployment as well? @ldb

Comment (ldb) [reply linkerd-viz/values.yaml]: Resolved 👍

**Review by isaschm — APPROVED** [review]
🙈

**Review by zuse168 — APPROVED** [review]
love it 💅🏻💅🏻💅🏻💅🏻

**Review by jgraeger — APPROVED** [review]
Assume it *would look good to me*

**Pull request merged**